bodhi/server/consumers/updates.py

```diff
@@ -55,7 +54,8 @@ class UpdatesHandler(fedmsg.consumers.FedmsgConsumer):
     config_key = 'updates_handler'
 
     def __init__(self, hub, *args, **kwargs):
-        self.db_factory = models.get_db_factory()
+        engine = initialize_db(config)
+        self.db_factory = util.transactional_session_maker(engine)
 
         prefix = hub.config.get('topic_prefix')
         env = hub.config.get('environment')
```

Comment:
Ahh, I missed that. I'm inclined to file an issue and clean that up in a later commit. There's lots more cleanup that needs to get done that I didn't include in this PR :(